jac/jaclang/plugin/default.py

```diff
@@ -54,6 +54,13 @@
 class JacAccessValidationImpl:
     """Jac Access Validation Implementations."""
 
+    @staticmethod
+    @hookimpl
+    def elavate_root() -> None:
```

Comment:
Everything looks fine, just that it should be spelt `elevate_root`

Reply:
@amadolid ping on this.